src/elements/transformer/Transformer.ts

```diff
@@ -0,0 +1,20 @@
+import type { FieldNameToValueMap } from "../../plugin/fieldViews/helpers";
```

Comment:
We've two files here, `transform` and `transformer` – this one seems to hold helper types, perhaps this could be called `types` or similar?

Reply:
Yeah I thought about that, I also considered adding a type folder. Happy with either really.

Reply:
As am I 👍